app/controllers/api/v1/orders_controller.rb

```diff
@@ -24,7 +24,7 @@ def cancel_order
 
       def submit_order
         order = Order.find(params.require(:order_id))
-        authorize order
+        authorize(order)
```

Comment:
@eclarizio Isn't this missing the action,
authorize(order, :submit_order)

Reply:
I think it should be fine, because there is a `submit_order?` method in the policy.